cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-scaling.adoc

```diff
@@ -0,0 +1,259 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-scaling"]
+= Tutorial: Scaling an application
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-scaling
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2024-04-10
+
+== Scaling
+
+You can manually scale your pods or you can automatically scale your pods by using the Horizontal Pod Autoscaler (HPA). You can also scale your cluster nodes.
+
+=== Manual pod scaling
+
+You can manually scale the pods and nodes of your applications:
+
+* by changing your replicaset and deployment definition 
+* by using the command line
+* by using the web console
+* by changing your *deployment* definition in the `ostoy-frontend-deployment.yaml`
+
+This workshop starts by using only one pod for the microservice. By using one pod, the Kubernetes Replication Controller strives to keep one pod alive. You can also define pod autoscaling by using the link:https://docs.openshift.com/container-platform/latest/nodes/pods/nodes-pods-autoscaling.html[Horizontal Pod Autoscaler](HPA). The HPA is based on workload, and it expands past defined limits if needed.
+
+.Prerequisites
+
+* Create a cluster with the xref:../../cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-lab-overview.adoc[OSToy microservice deployed].
+* Download the link:https://www.rosaworkshop.io/ostoy/yaml/ostoy-microservice-deployment.yaml[ostoy-microservice-deployment.yaml].
```

Comment:
the microservice deployment manifest is not needed to be downloaded.  What is the prerequisites would be:
1. A ROSA Cluster
2. The OSTOY application successfully deployed.